src/components/remotecontrol/remotecontrol.js

```diff
@@ -110,31 +110,53 @@ define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageL
         return null;
     }
 
-    function updateNowPlayingInfo(context, state) {
+    function updateNowPlayingInfo(context, state, serverId) {
         var item = state.NowPlayingItem;
         var displayName = item ? getNowPlayingNameHtml(item).replace("<br/>", " - ") : "";
-        context.querySelector(".nowPlayingPageTitle").innerHTML = displayName;
+        console.debug(JSON.stringify(item, null, 4));
+        if (item.Type == "Audio" || item.MediaStreams[0].Type == "Audio") {
+            var songName = item.Name;
+            if (item.Album != null && (item.Artists != null)) {
+                var albumName = item.Album;
+                var artistName;
+                if (item.ArtistItems != null) {
+                    artistName = item.ArtistItems[0].Name;
+                    context.querySelector(".nowPlayingAlbum").innerHTML = '<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=' + item.AlbumId + '&amp;serverId=' + (item.ServerId || serverId) + '">' + albumName + '</a>';
+                    context.querySelector(".nowPlayingArtist").innerHTML = '<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=' + item.ArtistItems[0].Id + '&amp;serverId=' + (item.ServerId || serverId) + '">' + artistName + '</a>';
+                    context.querySelector(".contextMenuAlbum").innerHTML = '<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=' + item.AlbumId + '&amp;serverId=' + (item.ServerId || serverId) + '"><i class="actionsheetMenuItemIcon listItemIcon listItemIcon-transparent material-icons">album</i> View album</a>';
+                    context.querySelector(".contextMenuArtist").innerHTML = '<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=' + item.ArtistItems[0].Id + '&amp;serverId=' + (item.ServerId || serverId) + '"><i class="actionsheetMenuItemIcon listItemIcon listItemIcon-transparent material-icons">person</i> View artist</a>';
+                } else {
+                    context.querySelector(".nowPlayingAlbum").innerHTML = albumName;
+                    artistName = item.Artists;
+                    context.querySelector(".nowPlayingArtist").innerHTML = artistName;
+                }
+            }
+            context.querySelector(".nowPlayingSongName").innerHTML = songName;
+        } else {
+            context.querySelector(".nowPlayingPageTitle").innerHTML = displayName;
+        }
 
-        if (displayName.length > 0) {
+        if (displayName.length > 0 && item.Type != "Audio") {
             context.querySelector(".nowPlayingPageTitle").classList.remove("hide");
         } else {
             context.querySelector(".nowPlayingPageTitle").classList.add("hide");
         }
 
         var url = item ? seriesImageUrl(item, {
-            maxHeight: 300 * 2
```

Comment:
Why?

Reply:
I did some testing and I just forgot to put it back to 300 * 2. Will do in my next commit.